lang/python/python-libselinux/Makefile

```diff
@@ -0,0 +1,97 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+SRC_NAME:=libselinux
+PKG_NAME:=python-$(SRC_NAME)
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(SRC_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(SRC_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=ea5dcbb4d859e3f999c26a13c630da2f16dff9462e3cc8cb7b458ac157d112e7
+PKG_BUILD_DEPENDS:=PACKAGE_python-libselinux:python3 PACKAGE_python-libselinux:swig/host
+HOST_BUILD_DEPENDS:=libsepol/host pcre/host
+
+PKG_MAINTAINER:=Thomas Petazzoni <thomas.petazzoni@bootlin.com>
+PKG_LICENSE:=PUBLICDOMAIN
+PKG_LICENSE_FILES:=LICENSE
+PKG_CPE_ID:=cpe:/a:selinuxproject:libselinux
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
Do we really need a host-build of that as well? What is using it?

Reply:
Things seem to build without the host-build.mk, so I removed it with the last push. I am presently testing with a fresh build.